src/components/WalletImageSlider.tsx

```diff
@@ -0,0 +1,67 @@
+import { sliderImage } from "./SliderImageWallet";
+import SliderButton from "./SliderButton";
+
+type SliderImageWalletProps = {
+  sliderValue: number;
+  setSliderValue: React.Dispatch<React.SetStateAction<number>>;
+  sliderValueRounder: (number: number) => number;
+};
+
+export default function WalletImageSlider({
+  sliderValue,
+  setSliderValue,
+  sliderValueRounder,
+}: SliderImageWalletProps) {
+  const SliderValueIsEqualorGreaterThanIndex = (index: number) => {
+    if (sliderValue >= index) return true;
+  };
+
+  const handleEventChange = (event) => {
+    setSliderValue(event.target.value);
+  };
+
+  const filledWidthCalucation = () => {
+    return `${(sliderValue / (sliderImage.length - 1)) * 100}%`;
+  };
```

Comment:
I don't think this needs to be a function does it? Is there a reason why this was the choice?

Reply:
previous PR asked to extract to variable with more context on what it's performing

Reply:
Why specifically a function over just a variable?

Reply:
no particular reason, just used to the pattern of a function instead of variable. Turned function into variable passed as variable instead of function.